app/src/modules/settings/routes/flows/components/logs-sidebar-detail.vue

```diff
@@ -86,18 +107,14 @@ function onToggle(open: boolean) {
 		<v-pagination v-if="pagesCount > 1" v-model="page" :length="pagesCount" :total-visible="3" />
 	</sidebar-detail>
 
-	<LogsDrawer :revision-id="previewing?.id" :flow="props.flow" @close="previewing = null" />
+	<logs-drawer :close-drawer="() => (selectedRevision = null)" :flow="flow" :revision="selectedRevision"></logs-drawer>
```

Comment:
Let’s keep the event approach for `@close`

```suggestion
	<logs-drawer :flow="flow" :revision="selectedRevision" @close="selectedRevision = null" />
```

Reply:
done